cmd/seednode/api/api.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/api/logs"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"github.com/gorilla/websocket"
+)
+
+var log = logger.GetOrCreate("seednode/api")
+
+// Start will boot up the api and appropriate routes, handlers and validators
+func Start(port int, marshalizer marshal.Marshalizer) error {
+	ws := gin.Default()
+	ws.Use(cors.Default())
+
+	registerRoutes(ws, marshalizer)
+
+	return ws.Run(fmt.Sprintf(":%d", port))
```

Comment:
We need to specify the interface, just like in elrond-go node

Reply:
done